include/onnxruntime/core/session/experimental_onnxruntime_cxx_api.h

```diff
@@ -22,8 +22,8 @@
 namespace Ort::Experimental {
 
 struct Session : Ort::Session {
-  Session(Env& env, ORTCHAR_T* model_path, SessionOptions& options)
-      : Ort::Session(env, model_path, options){};
+  Session(Env& env, std::string model_path, SessionOptions& options)
```

Comment:
Good catch. I mostly develop on Linux so wasn't aware. I've changed it to `std::basic_string<ORTCHAR_T>&` which I think is cross-compatible with Windows and accounts for `wchar_t`.

Reply:
Note that you all already use `std::string` in [other parts](https://github.com/microsoft/onnxruntime/blob/master/include/onnxruntime/core/session/onnxruntime_cxx_api.h#L30) of the C++ API so I'm not sure why the unit tests pass on them if `std::string` really does cause problems. Perhaps some implicit conversion occurs in Windows which makes it okay.

Reply:
Yeah, the reason this one is ORTCHART_T is because it's a filename and on windows filenames are natively wchar_t. The other ones are internal so we can just keep them as std::string. Windows has added a UTF8 mode recently but we can't rely on it for all supported versions yet.